### ISSUE TITLE
just checking about changing event...

### ISSUE THREAD

**Issue by CoIin** [myflashlab/webView-ANE#89]
I'm a club member, but have not used the webView ANE because I need to detect changing events, like I can with StageWebView. It's vital to my needs!

If it turns out the documentation is wrong or misleading, and you can tell when the URL has changed (before the page begins to load), I have another question: Is it possible to clear the memory that the webView was using on a previous pages it is no longer showing? StageWebView seems to continue to use up memory after you have closed previous pages. In case it helps to be specific, I'm using an iframe as an attempt to encourage StageWebView to forget the old content, but still I get memory warnings from iOS.

**Comment by myflashlab** [comment]
When initializing the ANE, you should add the following listeners. They will let you know when a page has started to load and when it finishes.
```actionscript
_ex.addEventListener(RichWebViewEvent.PAGE_STARTED, onPageStarted);
_ex.addEventListener(RichWebViewEvent.PAGE_PROGRESS, onPageProgress);
_ex.addEventListener(RichWebViewEvent.PAGE_FINISHED, onPageFinished);
```

> tell when the URL has changed (before the page begins to load), 

so, maybe you can stop the page load before it starts? I am marking this as an enhancement but would you please give us a practical use-case for this?

> Is it possible to clear the memory that the webView was using on a previous pages it is no longer showing?

We haven't added anything special to do this job but it seems like the new page load removes memory consumption with the old pages automatically. we have never had a report on this. so I guess it's fine.

**Comment by CoIin** [comment]
We use custom URLs to tell the shell app to do things. The app can do a preventdefault to stop the page loading. Actually changing the page and trying to stop it from loading, might be visually glitchy. Another way to ask about this, if your ANE is a replacement to StageWebView, it ought to have the same features, including the location changing event. 

There are a lot of pages that talk about getting UIWebView to clear memory from a precious page, it's not an AIR specific issue. It would have been nice if you had found a solution to the memory issues.

**Comment by myflashlab** [comment]
Thanks for the information. we will look into this more and get ready for a next version release.

**Comment by CoIin** [comment]
Thanks for the very fast reply!

**Comment by myflashlab** [comment]
@CoIin We just released V6.3.0 which now you can truly stop prevent a url load. read the [changelog here].(https://github.com/myflashlab/webView-ANE#changelog)

About the memory leak after visiting several pages, would you please open a new ticket dedicated to that problem only? And while doing that, that would be super helpful if you can show us a URL where we can load pages and see the memory problem in action. Thanks.
